sdk/php/src/Connection/version.php

```diff
@@ -1 +1,3 @@
-<?php /* Code generated by dagger. DO NOT EDIT. */ return '0.12.4';
+<?php
+
+/* Code generated by dagger. DO NOT EDIT. */ return '0.12.4';
```

Comment:
This might produce a linting error when the version is bumped via the release process. Should update the code that generates this in `/dev/sdk_php.go` so that it doesn't produce a linting error.

Reply:
FTR, it just means you need to adjust the following to look like that (add two `\n` after `<?php`):

https://github.com/dagger/dagger/blob/0f8b9d58535fcf65d560448ce4e31dfebecdfdac/dev/sdk_php.go#L102-L103